### PR TITLE
[Misc] add print support for uint128

### LINKED ISSUE #1704: [CLI tool] `v128` return value is not handled
## Description
When executing the `func1` function in main.wasm, wasmer print `79228162514264337593543950336`, but wasmedge print nothing.
The main.wasm is uploaded as main.txt since it is not supported to upload a file end with .wasm.

[main.txt](https://github.com/WasmEdge/WasmEdge/files/9228709/main.txt)

### Current State
print nothing 

### Expected

print `79228162514264337593543950336`

## Environment

- Hardware Architecture: ( x86\_64)
- Operating system: (Ubuntu 20.04 and macOS 10) the problem occur both in ubuntu and macOS 

## Steps to Reproduce
In wasmer:  `wasmer --enable-all main.wasm -i func1`
In wasmedge : `wasmedge --reactor --enable-all main.wasm func1` or `wasmedgec main.wasm main-edge.wasm; wasmedge --reactor --enable-all main-edge.wasm func1`

### PR DESCRIPTION
**Pull request by LFsWang**
Fix #1704 

Signed-off-by: LFsWang <tnst92002@gmail.com>

### CODE REVIEW

File: lib/driver/runtimeTool.cpp [hunk 1]
```diff
@@ -22,6 +22,21 @@
 #include <vector>
 
 namespace WasmEdge {
+std::ostream &operator<<(std::ostream &OS, uint128_t Value) {
+  std::string S;
+  if (Value < (uint64_t)-1) {
```
Comment (q82419) [inline lib/driver/runtimeTool.cpp:27]: Use the `std::numeric_limits<uint64_t>::max`

Comment (q82419) [reply lib/driver/runtimeTool.cpp:27]: `<=`, not `<`

File: lib/driver/runtimeTool.cpp [hunk 2]
```diff
@@ -22,6 +22,21 @@
...
+    OS << uint64_t(Value);
+    return OS;
+  }
+
+  while (Value != 0) {
+    S.push_back('0' + Value % 10);
```
Comment (q82419) [inline lib/driver/runtimeTool.cpp:33]: It's proper to separate the `uint128_t` into 2 uint64_t to print, rather than construct them into a string.

**Review by q82419 — CHANGES_REQUESTED** [review]

File: lib/driver/runtimeTool.cpp (hunk 1 above)
Comment (LFsWang) [reply lib/driver/runtimeTool.cpp]: fixed

File: include/common/int128.h [hunk 3]
```diff
@@ -14,17 +14,20 @@
 #pragma once
 
 #if defined(__x86_64__) || defined(__aarch64__)
+#include <ostream>
```
Comment (hydai) [inline include/common/int128.h:17]: Looks like the `<ostream> is contained in both `if` and `else` conditions. Maybe we can move this include out?

File: lib/common/int128.cpp [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+#include <algorithm>
+#include <cstdint>
+#include <iomanip>
+#include <limits>
+#include <ostream>
+#include <sstream>
+
+namespace WasmEdge {
+static inline uint128_t Mod(uint128_t LHS, uint128_t RHS) noexcept;
```
Comment (hydai) [inline lib/common/int128.cpp:14]: The function name should be `lower cases`.

**Review by hydai — CHANGES_REQUESTED** [review]

**Review by hydai — APPROVED** [review]

**Review by q82419 — APPROVED** [review]

**Pull request merged**